### PR TITLE
Heatmap bug fixed

### PR DESCRIPTION
**Pull request by chenjiandongx**
本次 PR 内容，

* 修复 #572 HeatMap 图纵坐标显示索引值，而非 data 值的 bug
* 文档更新

### CODE REVIEW

File: pyecharts/echarts/option.py [hunk 1]
```diff
@@ -487,7 +487,8 @@ def label_formatter(params):
     if yaxis_force_interval is not None:
         _yAxis["interval"] = yaxis_force_interval
 
-    return [_xAxis], [_yAxis]
+    # 返回字典
+    return [_xAxis.config], [_yAxis.config]
```
Comment (chenjiandongx) [inline pyecharts/echarts/option.py]: @chfw here

Comment (chfw) [reply pyecharts/echarts/option.py]: pyecharts-javascription will return config: https://github.com/pyecharts/pyecharts-javascripthon/blob/master/pyecharts_javascripthon/api.py#L106, that's why I am not seeing a fix.

Comment (chfw) [reply pyecharts/echarts/option.py]: it means the user has used old version of pyecharts-javascription

Comment (chenjiandongx) [reply pyecharts/echarts/option.py]: https://github.com/pyecharts/pyecharts/blob/master/pyecharts/charts/heatmap.py#L66-L71

这两个语句是没有效果的，这也是导致该 issue 的原因

Comment (chfw) [reply pyecharts/echarts/option.py]: OK, bug is here: https://github.com/pyecharts/pyecharts/blob/master/pyecharts/echarts/json_serializable.py#L16

Comment (chfw) [reply pyecharts/echarts/option.py]: self._config should have been used

Comment (chfw) [reply pyecharts/echarts/option.py]: my bad

Comment (chenjiandongx) [reply pyecharts/echarts/option.py]: 如果你有更好的解决方案也可以提出来，不然你可能就得修改一下 pyecharts-javascription 的代码了

Comment (chfw) [reply pyecharts/echarts/option.py]: 稍等，我来改一下你这个branch

**Pull request merged**